Makefile

```diff
@@ -293,36 +297,37 @@ LINUX_INCLUDES = include/linux/futex.h include/linux/version.h
 
 CRT_TICKET_LIBS = $(addprefix lib/,$(notdir $(CRT_TICKETS)))
 STATIC_TICKET_LIBS = lib/libc_repo.a
-ALL_TICKET_LIBS = $(CRT_TICKET_LIBS) $(STATIC_TICKET_LIBS) $(TEXTUAL_DB) $(DB)
+ALL_TICKET_LIBS = $(CRT_TICKET_LIBS) $(STATIC_TICKET_LIBS) $(REPOFILE)
```

Comment:
Should ALL_TICKET_LIBS include  `$(TEXTUAL_DB)` not  `$(REPOFILE)`?

Reply:
It shouldn’t include $(TEXTUAL_DB) (as mentioned [above](https://github.com/SNSystems/musl-prepo/pull/14#issue-1014962447)). There’s no need for the export file to be created here since, if necessary, it can be done later by simply running pstore-export on the database.

It’s likely that $(REPOFILE) is also not necessary, but I haven’t checked that.

Reply:
Commit 2150d90 removes TEXTUAL_DB altogether.